src/listener.c

```diff
@@ -264,6 +264,18 @@ accept_listener_protocol(struct Listener *listener, char *protocol) {
     return 1;
 }
 
+#ifdef SO_REUSEPORT
+int
+accept_listener_reuseport(struct Listener *listener, char *reuseport) {
+    if (strncasecmp(reuseport, "yes", strlen(reuseport)) == 0)
```

Comment:
I had to add the function to the listener.c because the listener is included by config and not vice versa... Maybe it's not the best solution.